docs/upstream-updates/index.rst

```diff
@@ -13,24 +13,30 @@ Rarely, you may also have to add or update an override.
 Creating a Schema File
 ----------------------
 
-First, you will need to fetch SchemaGen_ and a copy of the Juju source.
-Once your copy of the Juju source is at the version you want to update to
-(probably the `develop` branch, or a release tag) and you have updated
-and reinstalled SchemaGen to reflect those changes, you just need to send
-the output into a file in the libjuju repository:
+Firstly, you'll need to checkout the Juju_ project and correctly have it
```

Comment:
I tend to avoid abbreviations like "you'll", "won't", "shouldn't" etc in docs and formal communication, because it is analogous to slurring. Pedantry on my part does not block landing though :)